documentation/final-deliverables/pdf-report.md

```diff
@@ -105,16 +105,55 @@ FR23. To "unmatch" Mentees and Mentors, the Mentee can communicate with the Ment
 
 ## Technical Specifications
 
- (needs to be added)
+**Overview:**
+
+Although the decision for our chosen tech stack was ultimately up to us, our client has technical experience, and so she offered recommendations for front-end and back-end libraries back in September. As such, we aimed to align with the client’s technical suggestions, as well as to pick an MVC model which was well documented. Ultimately, this resulted in React being used for our front-end, Node for our back-end, and Firebase for our database.
```

Comment:
I don't think React uses the MVC model. From [their doc](https://reactjs.org/blog/2013/06/05/why-react.html): "React isn’t an MVC framework."
```suggestion
Although the decision for our chosen tech stack was ultimately up to us, our client has technical experience, and so she offered recommendations for front-end and back-end libraries back in September. As such, we aimed to align with the client’s technical suggestions, as well as to pick well-documented technologies. Ultimately, this resulted in React being used for our front-end, Node for our back-end, and Firebase for our database.
```

Unless you mean that the combo of these three things is MVC?

Reply:
Yea, my thinking was that the combination of React, Node, and Firebase would follow an MVC model, but my logic could be flawed! I'll commit your suggestion :)